### DISCUSSION TITLE
conda install

### DISCUSSION THREAD

**Discussion by arobertsonithree** [moj-analytical-services/splink#1615] [General]
Installing via conda install -c conda-forge splink didn't provide from splink.duckdb.blocking_rule_library or from splink.datasets

No solution using update either.

(pip install to a different environment and copying across worked)

**Comment by ADBond** [comment]
What version of Splink do you get when you install via `conda`? Do you have other dependencies in the environment? Could be that some other dependency provides some constraint that means you end up with an older version of `splink` (i.e. one before `splink.duckdb.blocking_rule_library` or `splink.datasets`).

**Reply by arobertsonithree** [reply]
thanks for responding so quickly. I had v3.9.7 but pandas 1.5.3 so guess having such an old version could have been the issue?
